### PR TITLE
2057-places-filter

### PR DESCRIPTION
**Pull request by asogwachiuba**
Place filter location is responsive as expected.
Simulator Screen Shot - iPhone SE (3rd generation) - 2023-06-01 at 12 11 27
Simulator Screen Shot - iPhone SE (3rd generation) - 2023-06-01 at 12 11 32

### CODE REVIEW

File: lib/core/services/places/place_services.dart [hunk 1]
```diff
@@ -222,9 +222,10 @@ class PlacesServicesImpl extends PlaceServices {
       if (category != null) "category": category,
       if (minPrice != null) "minPrice": minPrice,
       if (maxPrice != null) "maxPrice": maxPrice,
+      if (amenities != null) "amenities": amenities,
     };
     selectedDays?.forEach((element) {
-      request.addAll({element: "8:30AM-R11:30PM"});
+      request.addAll({element: "8:30AM-11:30PM"});
```
Comment (divincoder) [inline lib/core/services/places/place_services.dart:228]: what does this have to do with this ticket.

Also, why is this hardcoded

**Review by divincoder — CHANGES_REQUESTED** [review]

File: lib/core/services/places/place_services.dart (hunk 1 above)
Comment (asogwachiuba) [reply lib/core/services/places/place_services.dart:228]: It specifies the time for any selected day. Its hard coded cause its general for all day

File: lib/features/events/views/eventlist/event_List_view_model.dart [hunk 2]
```diff
@@ -252,7 +252,7 @@ class EventListViewModel extends IbloovViewModel {
 
     result.when(success: (status) {
       if (status.categories != null) {
-        categories.add(Category(sId: "0", name: "All"));
+        categories.add(EventCategory(sId: "0", name: "All"));
```
[the remark this answers is not in the document]
Comment (asogwachiuba) [reply lib/features/events/views/eventlist/event_List_view_model.dart:255]: I followed the refactoring done to the eventlist on the release branch.

[the remark this answers is not in the document]
Comment (divincoder) [reply lib/features/events/views/eventlist/event_List_view_model.dart:255]: Hmm, why?

The release branch will be merged with develop. If there was sth there you needed. Shouldn't you create a branch to update develop with release than reImplementing sth done there?

I don't trust the changes in this PR. Especially comparing to the description.

Please do another PR. I will merge release to develop soon.
